india_compliance/patches/v14/migrate_and_update_tds_section_as_per_income_tax_act_2025.py

```
@@ -2,7 +2,12 @@
 from frappe.query_builder import Case
 from frappe.query_builder.functions import IfNull
 
-from india_compliance.income_tax_india.constants import OLD_TDS_SECTIONS, TDS_ENTITY_TYPE
+from india_compliance.income_tax_india.constants import (
```

Flake8 (7.3.0)

[error] 'india_compliance.income_tax_india.constants.NEW_TDS_SECTION' imported but unused

(F401)